bodhi/server/consumers/masher.py

```diff
@@ -902,6 +903,36 @@ def compose_atomic_trees(self):
         mashed_repos = dict([('-'.join(os.path.basename(repo).split('-')[:-1]), repo)
                              for repo in self.state['completed_repos']])
         for tag, mash_path in mashed_repos.items():
+            #  TODO: Find out the actual values for the tag
+            #  also for the code below, should we use the mash_path instead
+            #  of current_workdir?
+            #  * Pungi config repos hsa $arch, does it matter in bodhi?
+            #  * Do we need any other config files in the pungi-fedora config directory before we fireup the pungi
+            if tag.find('f26') != -1:  # This is a F26 build
+                current_workdir = os.path.join(self.path, self.id)
+                repo_dir = os.path.join(current_workdir, "pungi-fedora")
+                branch = "f26"
+                git_checkout(repo_dir, "https://pagure.io/pungi-fedora", branch)
+                # Prepare the Repo URL
+                dirpath = os.path.join(current_workdir, tag)[9:]
+                url = "https://kojipkgs.fedoraproject.org/{0}/$arch/".format(dirpath)
+                pungi_config_file = os.path.join(repo_dir, "fedora-atomic.conf")
+                with open(pungi_config_file) as fobj:
+                    lines = fobj.readlines()
+
+                idx = 0  # The place to insert the new repo path
+                for i, line in enumerate(lines):
+                    if line.find('"source_repo_from": [ "Cloud",') != -1:
+                        idx = i
+                        break
+                if idx != 0:
+                    lines.insert(idx + 1, url)
```

Comment:
Why not use a string replace or a regex here with a ```\n```? This way you don't need to read through the file, find an index, and do all this machinery. Optional.